e2e/ios/E2EOktaReactNative/AppDelegate.mm

```diff
@@ -30,10 +30,10 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
 - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
 {
-  return [self getBundleURL];
+  return [self bundleURL];
 }
 
-- (NSURL *)getBundleURL
+- (NSURL *)bundleURL
```

Comment:
These changes are from react-native upgrade helper for versions 0.73 -> 0.74. The e2e app doesn't build without those changes